nss-fumbling-penguins/nutshell
# Rm api manager

## Link to Ticket
Closes # [9]()

## Description of Proposed Changes

Added an api manager with methods to get information from the database.

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <feature_branch>

```

1. check function syntax.
1. start up api on port 8088
1. test out functions by adding them in the main.js file, console logging results, and refreshing the page
1. functions should either manipulate the api or print out data to the console


## Impacted Areas in Application

List general components of the application that this PR will affect:

API

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
